lib/remote.go

```diff
@@ -37,26 +43,12 @@ type FeedsParams struct {
 
 // Feeds returns a listing of datasets from a number of feeds like featured and
 // popular. Each feed is keyed by string in the response
-func (r *RemoteMethods) Feeds(ctx context.Context, p *FeedsParams) (map[string][]dsref.VersionInfo, error) {
-	if r.inst.http != nil {
-		res := map[string][]dsref.VersionInfo{}
-		err := r.inst.http.Call(ctx, AEFeeds, p, &res)
-		if err != nil {
-			return nil, err
-		}
-		return res, nil
+func (m RemoteMethods) Feeds(ctx context.Context, p *FeedsParams) (map[string][]dsref.VersionInfo, error) {
+	got, _, err := r.d.Dispatch(ctx, dispatchMethodName(r, "feeds"), p)
```

Comment:
`r.d.Dispatch(...` does not work should be `m.d.Dispatch(...` if I got this right.

Reply:
yea, i squashed a commit and i guess did it wrong, and it lost this round of work :/